packages/ra-ui-materialui/src/button/LocalesMenuButton.tsx

```diff
@@ -48,6 +48,7 @@ export const LocalesMenuButton = (props: LocalesMenuButtonProps) => {
         <Root component="span">
             <Button
                 color="inherit"
+                variant="text"
```

Comment:
This is to avoid an outline when the theme sets the button variant to `outlined`